app/helpers/fraud_indicators_helper.rb

```diff
@@ -11,11 +11,21 @@ def link_to_indicator_list(fraud_indicator)
              return nil
            end
     table = fraud_indicator.list_model_name.split("::").last.downcase
+    path_with_type = "hub_#{type}_#{table}s_path"
+    path_without_type = "hub_#{table}s_path"
 
-    if respond_to?("hub_#{type}_#{table}s_path")
-      link_to send("hub_#{type}_#{table}s_path") do
-        content_tag :i, "list_alt", class: "icon-"
-      end
-    end
+    matching_path = nil
```

Comment:
the timezones page isn't prefixed with risky vs safe, so this needed a bit of a logic update